package.json

```diff
@@ -22,10 +22,8 @@
   },
   "devDependencies": {
     "@tsconfig/node18": "^18.2.0",
```

Comment:
CI Test is failing due this TS dep is blocking the ci (LOL):
```
{
  "devDependencies": {
    "@types/node": "^18.0.0"
  }
}
```

By default `ubuntu` uses node20 atm: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Since we are on node20, we can remove or update it (idk)

https://github.com/fastify/action-pr-title/blob/main/action.yml#L33